x/concentrated-liquidity/types/params.go

```diff
@@ -46,6 +48,7 @@ func DefaultParams() Params {
 			"ibc/0CD3A0285E1341859B5E86B6AB7682F023D03E97607CCC1DC95706411D866DF7", // DAI
 			"ibc/D189335C6E4A68B513C10AB227BF1C1D38C746766278BA3EEB4FB14124F1D858", // USDC
 		},
+		IsPermissionlessPoolCreationEnabled: false,
```

Comment:
Can we please make this true for the default? I can imagine with making custom testnets / state exports, this is just going to be one more thing we have to always change, whereas we know we are setting this to true in the upgrade handler which would just be a one time thing.

Reply:
I think this is actually preferable to keep this consistent with mainnet so that for out v16 testnets we are testing pool creation as disabled. This also helps to run e2e by default with CL pool creation disabled which is intended.

Once we change the param on mainnet, we can flip this flag here as well.

Let me know if that makes sense

Reply:
Spoke offline on this. While I personally think the overhead of changing this for localosmosis and testnetify is greater than any risk this would expose, will not block on my opinion!

Reply:
I will think more about this tomorrow. Will likely go with your suggestion

Reply:
I actually agree with @p0mvn on keeping this flag as what is going to be in mainnet. For localosmosis, if necessary, we can  add a line to edit the genesis manually to have the param to true, for testnet, we can probably change it using gov proposals?

Reply:
Updated localosmosis config: [8205f7d](https://github.com/osmosis-labs/osmosis/pull/4952/commits/8205f7da85be0bd8f951ddba493662782c62c37f)

going to go with the flag as on mainnet. We can change this later if ends up being problematic